src/cpio/client_providers/kms_client_provider/azure/azure_kms_client_provider_utils.cc

```diff
@@ -195,38 +196,38 @@ AzureKmsClientProviderUtils::GenerateWrappingKey() {
 /**
  * @brief Convert a private PEM wrapping key to EVP_PKEY using private key
  *
- * @param wrappingPemKey RSA PEM key used to wrap a key.
+ * @param wrapping_pem_key RSA PEM key used to wrap a key.
  * @return std::shared_ptr<EvpPkeyWrapper> containing the EVP_PKEY.
  */
-std::shared_ptr<EvpPkeyWrapper> AzureKmsClientProviderUtils::GetPrivateEvpPkey(
-    std::string wrappingPemKey) {
+absl::StatusOr<std::shared_ptr<EvpPkeyWrapper>>
+AzureKmsClientProviderUtils::GetPrivateEvpPkey(std::string wrapping_pem_key) {
   ERR_clear_error();
 
   // Create a BIO wrapper to manage memory
-  BIOWrapper bioWrapper;
+  BIOWrapper bio_wrapper;
 
   // Ensure the bio is created successfully before using it
-  if (bioWrapper.get() == nullptr) {
-    throw std::runtime_error("Failed to create BIO");
+  if (bio_wrapper.get() == nullptr) {
+    return absl::InternalError("Failed to create BIO");
   }
 
   // Write the PEM key data into the BIO
-  if (BIO_write(bioWrapper.get(), wrappingPemKey.c_str(),
-                wrappingPemKey.size()) <= 0) {
-    throw std::runtime_error(
+  if (BIO_write(bio_wrapper.get(), wrapping_pem_key.c_str(),
+                wrapping_pem_key.size()) <= 0) {
+    return absl::InternalError(
         "Failed to write PEM data to BIO in GetPrivateEvpPkey");
   }
 
   // Read the private key from the BIO
-  EVP_PKEY* pkey = PEM_read_bio_PrivateKey(bioWrapper.get(), NULL, NULL, NULL);
+  EVP_PKEY* pkey = PEM_read_bio_PrivateKey(bio_wrapper.get(), NULL, NULL, NULL);
```

Comment:
const?
Also several in this file

Reply:
With the almost same reason as bio_wrapper, I think we can leave it.

If you just change it to const, it fails to compile when you pass it to the wrapper of EVP_PKEY.